packages/@ourworldindata/grapher/src/scatterCharts/ScatterPlotChart.tsx

```diff
@@ -1444,7 +1443,7 @@ export class ScatterPlotChart
                 this.transformedTable.getColorForEntityName(entityName)
             if (keyColor !== undefined) series.color = keyColor
             else if (!this.colorColumn.isMissing) {
-                const colorValue = last(
+                const colorValue = R.last(
                     series.points.map((point) => point.color)
                 )
```

Comment:
Sorry, this is not related to your change, but I just realized that we can trivially optimize this code:
```suggestion
                const colorValue = R.last(series.points)?.color
```